src/OrganisationRegistry.KboMutations/KboMutationsModule.cs

```diff
@@ -7,18 +7,12 @@ namespace OrganisationRegistry.KboMutations
 
     public class KboMutationsModule : Module
     {
-        private readonly IConfiguration _configuration;
-        private readonly IServiceCollection _services;
-
         public KboMutationsModule(
             IConfiguration configuration,
             IServiceCollection services)
         {
-            _configuration = configuration;
-            _services = services;
-
-            _services.Configure<KboMutationsConfiguration>(
```

Comment:
I have some doubts regarding the necessity of this class. Maybe inline this in the calling code?

Reply:
done